### ISSUE TITLE
Locked TFT different from the Dashboard

### ISSUE THREAD

**Issue by A-Harby** [threefoldtech/grid_weblets#1238]
### Description

Locked TFT displayed in the dashboard is 0, while the locked TFT in the weblets is 65 TFT.

### Logs/Alerts

![image](https://user-images.githubusercontent.com/109473873/212042719-b5dae0db-42bb-44c3-959f-0f104f8dc603.png)
![image](https://user-images.githubusercontent.com/109473873/212042781-c1833207-2da7-4191-ad13-e6d743a0333a.png)

**Comment by xmonader** [comment]
which environment?

**Comment by zaelgohary** [comment]
![image](https://user-images.githubusercontent.com/40770501/212049518-ca4f65c4-82dd-4b7f-a64c-0a61eccf6bc0.png)

![image](https://user-images.githubusercontent.com/40770501/212049620-aeb179aa-aa94-4b2a-b763-0d809111f599.png)

This is my balance from both Dashboard & Weblets locally. This issue is fixed [here](https://github.com/threefoldtech/tfgrid_dashboard/issues/418l). I guess it's not reflected as devnet hasn't been released since then.

**Comment by A-Harby** [comment]
This is true, [The locked TFT issue](https://github.com/threefoldtech/tfgrid_dashboard/issues/416) was solved and verified, but apparently for some reason some of the solved issue are re-appearing again like [Failed to create twin](https://github.com/threefoldtech/tfgrid_dashboard/issues/456) which was solved in [issue](https://github.com/threefoldtech/tfgrid_dashboard/issues/415), and  [Version environment variable](https://github.com/threefoldtech/tfgrid_dashboard/issues/455) which was solved in [issue](https://github.com/threefoldtech/tfgrid_dashboard/issues/284).

I think this is a result of an outdated version being re-released again.
